kaiax/gov/impl/api.go

```diff
@@ -59,6 +59,28 @@ func (api *KaiaAPI) GetParams(num *rpc.BlockNumber) (gov.PartialParamSet, error)
 	return getParams(api.g, num)
 }
 
+func distill(g *GovModule, gp gov.ParamSet, blockNum uint64) gov.ParamSet {
```

Comment:
```suggestion
func patchDeprecatedParams(gp gov.ParamSet, rule params.Rules) gov.ParamSet {
```
I guess "distill" is somewhat too generic? Also passing GovModule just for chain config is too much. It's difficult to write unit tests.

Reply:
done https://github.com/kaiachain/kaia/pull/174/commits/382ced779817c68b7a25ea2949e574c80b717087